feat: adapt code and playgrounds to latest changes on ecosystem

alvarosabu opened the pull request:
- Updated @tresjs/cientos and @tresjs/core to version 5.0.0-next.2 and 5.0.0-next.3 respectively in package.json and pnpm-lock.yaml.
- Refactored GLTF model loading in BlenderCube.vue and Ducky.vue to use state instead of scene for better reactivity.
- Removed unnecessary Suspense components in DepthOfFieldDemo.vue and DotScreenDemo.vue for improved performance.
- Updated ESLint auto-imports and components.d.ts for better type support.
- Cleaned up unused package.json file in playground/public/nuxt-stones.

alvarosabu committed:
feat: adapt code and playgrounds to latest changes on ecosystem

- Updated @tresjs/cientos and @tresjs/core to version 5.0.0-next.2 and 5.0.0-next.3 respectively in package.json and pnpm-lock.yaml.
- Refactored GLTF model loading in BlenderCube.vue and Ducky.vue to use state instead of scene for better reactivity.
- Removed unnecessary Suspense components in DepthOfFieldDemo.vue and DotScreenDemo.vue for improved performance.
- Updated ESLint auto-imports and components.d.ts for better type support.
- Cleaned up unused package.json file in playground/public/nuxt-stones.

alvarosabu committed:
refactor: remove unused watchEffect import in BlenderCube.vue

- Cleaned up the imports in BlenderCube.vue by removing the unused watchEffect import, improving code clarity and maintainability.

alvarosabu committed:
chore: update @tresjs/core to version 5.0.0-next.5

- Upgraded @tresjs/core in package.json and playground/package.json to version 5.0.0-next.5 for improved functionality and compatibility.
- Updated pnpm-lock.yaml to reflect the new version of @tresjs/core and its dependencies.
- Refactored render logic in EffectComposerPmndrs.vue and EffectComposer.vue to utilize notifySuccess callback for better rendering control.

Tinoooo commented on package.json:
**issue:** peer dependency must be bumped.

Tinoooo commented on src/core/three/EffectComposer.vue:
```diff
@@ -47,22 +47,19 @@ watchEffect(() => {
 
-render(() => {
-  if (renderCtx.frames.value > 0 && effectComposer.value && props.enabled) {
+render((notifySuccess) => {
```
**issue(maybe):** In case we merge https://github.com/Tresjs/tres/pull/1044, this is not up to date anymore.

Tinoooo reviewed (changes requested):
I provided an [MR](https://github.com/Tresjs/post-processing/pull/208) that addresses the comments I made.

Tinoooo commented:
**question:** Shall we introduce a unified major version number to make it more obvious that the package is compatible with core v5?

Tinoooo committed:
feat: adapt to core v5 changes 2 in MR 1043 (#208)

* refactor: streamline rendering logic in effect processors

* chore: update dependencies and improve rendering logic in on-demand playground example

Tinoooo commented on package.json:
**question:** Shall we introduce a unified major version number to make it more obvious that the package is compatible with core v5?

Tinoooo replied:
We decided to postpone to later as there are thoughts about a mono-repo already.

alvarosabu committed:
chore: remove unused onControlChange function in on-demand.vue

- Eliminated the onControlChange function as it was no longer necessary, simplifying the component's logic.
- Updated OrbitControls to remove the change event listener, streamlining the rendering process.

alvarosabu committed:
chore: update @tresjs/core to version 5.0.0-next.6

- Upgraded @tresjs/core in package.json and playground/package.json to version 5.0.0-next.6 for enhanced features and stability.
- Updated pnpm-lock.yaml to reflect the new version and its dependencies, ensuring compatibility across the project.

alvarosabu committed:
Merge branch 'next' into feat/adapt-to-core-v5-changes
